shared/references/cce-redhat-avail.txt

```diff
@@ -1,8 +1,5 @@
-CCE-83392-1
```

Comment:
These CCEs from line 1 and 2 seem to be from a different piece of work. They are not included in the new rule.

Reply:
You are right, I fixed it. Thank you.